src/sentry/models/prompts_activity.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.conf import settings
+from django.utils import timezone
+from jsonfield import JSONField
+
+from sentry.db.models import (BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr)
+
+
+class PromptsActivity(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    project_id = BoundedPositiveIntegerField(db_index=True)
```

Comment:
Let's do null=True here and for organization_id (to support org-level and user-level features)